### PR TITLE
Preparation of release 2.8.3

### PR DESCRIPTION
**Pull request by vkatsuba**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
 erGW - 3GPP GGSN and PDN-GW in Erlang
 =====================================
 
+Version 2.8.3 - 21 January 2021
+-------------------------------
+
+* Add global `node_id` config setting and use it in `PFCP` - [PR #278](https://github.com/travelping/ergw/pull/278)
+* Replace context key tupels with records - [PR #279](https://github.com/travelping/ergw/pull/279)
```
Comment (vkatsuba) [inline NEWS.md:8]: Fix typo

**Review by fholzhauser — APPROVED** [review]

**Comment by mgumz** [comment]
please order change-items by "features, improvements, bugs, other"

File: NEWS.md [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+* Fix online charging `URR` link - [PR #282](https://github.com/travelping/ergw/pull/282)
+* Configurable retry timings for `SMF` / `UPF` heartbeats - [PR #283](https://github.com/travelping/ergw/pull/283)
+* Add missing decription of `PFCP` metrics - [PR #285](https://github.com/travelping/ergw/pull/285)
+* Added `PFCP` `Sx` association metric - [PR #286](https://github.com/travelping/ergw/pull/286)
+* Update [ergw_aaa](https://github.com/travelping/ergw_aaa) tag to [3.6.6](https://github.com/travelping/ergw_aaa/releases/tag/3.6.6) - [PR #287](https://github.com/travelping/ergw/pull/287)
+* Remove dotfiles - [PR #289](https://github.com/travelping/ergw/pull/289)
+* Remove diagrams - [PR #290](https://github.com/travelping/ergw/pull/290)
```
Comment (mgumz) [inline NEWS.md:15]: combine as "cosmetics" or something like that.

File: NEWS.md [hunk 3]
```diff
@@ -1,6 +1,23 @@
...
+Version 2.8.3 - 21 January 2021
+-------------------------------
+
+* Add global `node_id` config setting and use it in `PFCP` - [PR #278](https://github.com/travelping/ergw/pull/278)
+* Replace context key tupels with records - [PR #279](https://github.com/travelping/ergw/pull/279)
+* Fix online charging `URR` link - [PR #282](https://github.com/travelping/ergw/pull/282)
+* Configurable retry timings for `SMF` / `UPF` heartbeats - [PR #283](https://github.com/travelping/ergw/pull/283)
+* Add missing decription of `PFCP` metrics - [PR #285](https://github.com/travelping/ergw/pull/285)
+* Added `PFCP` `Sx` association metric - [PR #286](https://github.com/travelping/ergw/pull/286)
+* Update [ergw_aaa](https://github.com/travelping/ergw_aaa) tag to [3.6.6](https://github.com/travelping/ergw_aaa/releases/tag/3.6.6) - [PR #287](https://github.com/travelping/ergw/pull/287)
```
Comment (mgumz) [inline NEWS.md:13]: can you please add some of the ergw_aaa related changes in here as well? its getting otherwise a bit hard to follow

Comment (vkatsuba) [reply NEWS.md:13]: Due to the fact that these are different projects, I left a link to notes of `ergw_aaa` release. I'm not sure if we should mix the notation of two different projects.

File: NEWS.md (hunk 1 above)
Comment (vkatsuba) [reply NEWS.md]: Fixed

File: NEWS.md (hunk 2 above)
Comment (vkatsuba) [reply NEWS.md]: Done as - **Refactoring** :bulb:

**Comment by vkatsuba** [comment]
> please order change-items by "features, improvements, bugs, other"

Good point, please review.

**Comment by mgumz** [comment]
allright. now the lines are prefixed with "feature", "bug" etc. now please also order them.
